tests/cases/fourslash/autoImportNonModules.ts

```diff
@@ -3,24 +3,20 @@
 // @Filename: /tsconfig.json
 //// {
 ////  "compilerOptions": {
-////    "module": "none",
+////    "module": "none"
 ////  }
 ////}
 
 // @Filename: /src/dirA/index.ts
 //// export * from "./thing1A";
-//// export * from "./thing2A";
 
 // @Filename: /src/dirA/thing1A.ts
 //// export class Thing1A {}
 //// Thing/**/
 
-// @Filename: /src/dirA/thing2A.ts
-//// export class Thing2A {}
-
 verify.completions({
   marker: "",
-  excludes: ["Thing2A"],
+  includes: ["!Thing1A"],

```

Comment:
🤔 What is this?

Reply:
> 🤔 What is this?

haha just trying the same thing to exclude it by including it with "!" , read it somewhere if exclude is giving error try including with "!".